data/859/073/99/85907399.geojson

```diff
@@ -92,7 +92,7 @@
     "wof:lang":[
         "ara"
     ],
-    "wof:lastmodified":1582326292,
+    "wof:lastmodified":1627493447,
```

Comment:
Nit: If nothing about the file is changing (besides the timestamp), then it shouldn't be included in the PR.

Reply:
I agree.. this must have slipped in during a round of `mz:` property updates. I'll try to root out similar records and reset them so they're not included in this PR's diff.

Reply:
This record (along with a few dozen others) was reverted to its original state fixed in 67fdf8c.